qiskit/providers/aer/utils/noise_model_inserter.py

```diff
@@ -48,20 +48,19 @@ def insert_noise(circuits, noise_model, transpile=False):
         else:
             transpiled_circuit = circuit
         qubit_indices = {bit: index for index, bit in enumerate(transpiled_circuit.qubits)}
-        result_circuit = circuit.copy(name=transpiled_circuit.name + '_with_noise')
+        result_circuit = transpiled_circuit.copy(name=transpiled_circuit.name + '_with_noise')
```

Comment:
This looks like a more important change.  Just glancing quickly, this new form certainly looks correct, though.

Reply:
Yeah, I know it would be better to create a separate PR for this bug, but I couldn't resist fixing it when I found it.